package.json

```diff
@@ -31,40 +31,29 @@
     "check:tailwind": "node ./scripts/check-tailwind.js"
   },
   "dependencies": {
-    "@edtr-io/core": "^2.8.2",
-    "@edtr-io/default-document-editor": "^2.5.0",
-    "@edtr-io/editor-ui": "^2.8.2",
-    "@edtr-io/math": "^2.8.2",
-    "@edtr-io/plugin": "^2.8.2",
-    "@edtr-io/plugin-anchor": "^2.8.2",
-    "@edtr-io/plugin-blockquote": "^2.8.2",
-    "@edtr-io/plugin-geogebra": "^2.8.2",
-    "@edtr-io/plugin-highlight": "^2.8.2",
-    "@edtr-io/plugin-image": "^2.8.2",
-    "@edtr-io/plugin-input-exercise": "^2.8.2",
-    "@edtr-io/plugin-multimedia-explanation": "^2.8.2",
-    "@edtr-io/plugin-rows": "^2.8.2",
-    "@edtr-io/plugin-sc-mc-exercise": "^2.8.2",
-    "@edtr-io/plugin-serlo-injection": "^2.8.3",
-    "@edtr-io/plugin-spoiler": "^2.8.2",
-    "@edtr-io/plugin-table": "^2.8.2",
-    "@edtr-io/plugin-text": "^2.8.2",
-    "@edtr-io/plugin-video": "^2.8.2",
-    "@edtr-io/renderer": "^2.8.2",
-    "@edtr-io/renderer-ui": "^2.8.2",
-    "@edtr-io/store": "^2.8.2",
-    "@edtr-io/ui": "^2.5.0",
-    "@fortawesome/fontawesome-svg-core": "^6.2.0",
+    "@fortawesome/fontawesome-svg-core": "^6.0.0",
     "@fortawesome/free-brands-svg-icons": "^6.2.0",
     "@fortawesome/free-regular-svg-icons": "^6.2.0",
-    "@fortawesome/free-solid-svg-icons": "^6.2.0",
+    "@fortawesome/free-solid-svg-icons": "^6.0.0",
+    "@fortawesome/react-fontawesome": "^0.1.0",
     "@ory/client": "^1.1.21",
     "@ory/integrations": "^1.1.0",
+    "@prezly/slate-lists": "^0.79.5",
     "@radix-ui/react-navigation-menu": "^1.1.1",
     "@serlo/authorization": "^0.47.0",
-    "@serlo/legacy-editor-to-editor": "Entkenntnis/tmp-serlo-legacy-editor-to-editor",
     "@serlo/markdown": "Entkenntnis/tmp-serlo-markdown",
     "@tippyjs/react": "^4.2.6",
+    "@types/algebra.js": "^0.2.2",
+    "@types/katex": "^0.11.1",
+    "@types/lodash": "^4.14.181",
+    "@types/ramda": "^0.28.0",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "@types/react-modal": "^3.13.1",
+    "@types/react-resizable": "^3.0.0",
+    "@types/react-syntax-highlighter": "^13.5.2",
+    "@types/shortid": "^0.0.29",
+    "@types/styled-components": "^5.0.0",
```

Comment:
Could be moved to dev dependencies.

Reply:
:+1: https://github.com/serlo/frontend/issues/2258